data/xml/2023.icnlsp.xml

```diff
@@ -0,0 +1,350 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2023.icnlsp">
+  <volume id="1" ingest-date="2023-12-31" type="proceedings">
+    <meta>
+      <booktitle>Proceedings of The sixth International Conference on Natural Language and Speech Processing (ICNLSP-2023)</booktitle>
```

Comment:
Weird capitalization, also title should be made consistent with previous volumes: 
<img width="1080" alt="image" src="https://github.com/acl-org/acl-anthology/assets/455256/d378a9a8-0787-458b-b328-8e21426710ee">

Please see [ingestion checklist](https://github.com/acl-org/acl-anthology/wiki/Ingestion-Checklist)

Reply:
Proceedings of the 6th International Conference on Natural Language and Speech Processing (ICNLSP-2023)